scarb-metadata/src/lib.rs

```diff
@@ -231,6 +231,9 @@ pub struct PackageMetadata {
     #[serde(flatten)]
     pub manifest_metadata: ManifestMetadata,
 
+    /// Compiler experimental features allowed for this package.
+    pub experimental_features: Option<serde_json::Value>,
```

Comment:
```suggestion
    pub experimental_features: Option<Vec<String>>,
```

I used `Vec` here instead of `HashSet`, to preserve ordering.

Reply:
So, do we want all of the fields (in `TomlManifest`, `Manifest` and `PackageMetadata`) to be vectors?

Reply:
discussed offline